nf_core/lint.py

```diff
@@ -251,11 +251,10 @@ def check_files_exist(self):
             os.path.join('.github', 'workflows', 'linting.yml')
         ]
         files_fail_ifexists = [
-            'Singularity',
+            'Singularity'
         ]
         files_warn_ifexists = [
-            '.travis.yml',
-            os.path.join('.circleci','config.yml')
```

Comment:
And here this was to warn people using CircleCI that the next version of tools might not support that any more

Reply:
I dont think we ever properly supported/recommended it anyway so should be fine to remove :+1:

Reply:
ah ok, perfect then, I thought Sarek only used CircleCI, but it's true it also uses GitHub actions, so no need to support CircleCI any more 👍